scene/resources/packed_scene.h

```diff
@@ -104,6 +110,8 @@ class SceneState : public RefCounted {
 		FLAG_ID_IS_PATH = (1 << 30),
 		TYPE_INSTANCED = 0x7FFFFFFF,
 		FLAG_INSTANCE_IS_PLACEHOLDER = (1 << 30),
+		FLAG_PATH_PROPERTY_IS_NODE = (1 << 30),
```

Comment:
I just noticed that `FLAG_ID_IS_PATH` and `FLAG_INSTANCE_IS_PLACEHOLDER` have the same value, and so does the new flag you added. Is this intended?

Reply:
Yeah they are for unrelated things